### ISSUE TITLE
Effekt test times dont add up

### ISSUE THREAD

**Issue by TheDying0fLight** [TheDying0fLight/FairSweeper#28]
![image](https://github.com/user-attachments/assets/23eacce6-68e9-4b92-9c15-cfa2f37619b3)

**Comment by jiribenes** [comment]
Yep, the total time is the time for all testing including the prints and everything (for the whole suite) as of right now, though I'm not sure how much of a good design that is...
Thanks for noticing though :)

**Comment by TheDying0fLight** [comment]
Ah thats what I thought could be the case. I don't know either but the times are useful anyways.
